awx/main/migrations/0153_instance_last_seen.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.20 on 2021-08-12 13:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0152_instance_node_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='instance',
+            name='last_seen',
+            field=models.DateTimeField(
+                editable=False,
+                help_text='Last time instance ran its heartbeat task for main cluster nodes. Last known connection to receptor mesh for execution nodes.',
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name='instance',
+            name='memory',
+            field=models.BigIntegerField(default=0, editable=False, help_text='Total system memory of this instance in bytes.'),
```

Comment:
Was wondering what changed here. Looks like it was just the `help_text`?

Reply:
yeah, exactly. None of these `Instance` fields have help text. I started out thinking I would write help_text for each one of them (since I was writing a migration anyway). But when I started on the first one, I realized that the values were wrong and filed https://github.com/ansible/ansible-runner/issues/788